src/main/resources/application.yml

```diff
@@ -1 +1,32 @@
+logging:
+  level:
+    root: INFO
+    org.springframework.web: INFO
+    org.springframework.security: INFO
+    org.springframework.security.oauth2: INFO
+    org.springframework.boot.autoconfigure: DEBUG
 
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          unity:
+            provider: hdf
+            client-id: helmholtz-marketplace
+            client-secret: ${secret:nopass}
```

Comment:
The necessary secret(s) should be explained in a README.md or in a separate developer documentation.

Reply:
Hmm... I'm kind of lost here, do you mean I need to explain what is client-secrete or how to obtain one?

Reply:
Maybe I misunderstood this :-) The secret has to be defined somewhere and for me it was not obvious where this should happen and maybe which other secrets also have to be defined for the application.

Reply:
[Here](https://auth0.com/docs/applications/concepts/client-secret#:~:text=A%20client%20secret%20is%20a,mobile%20or%20browser%2Dbased%20apps.) is a definition of client secret.

Generally speaking, when you set a value of application properties as set here, that is:
```yml
client-secret: ${secret:nopass}
``` 
It means the value maybe set at an environment variable, in this case, let assume you want to run the jar file in the command line and you want to supply the client-secret, you will typed something like this:

```bash
 java -jar -Dsecret=<put-the-seceret-here> helmholtz-marketplace-server-0.0.1-SNAPSHOT.jar
```

Hope this explanation clarify the mix-up